### ISSUE TITLE
incorrect possition within line

### ISSUE THREAD

**Issue by yohny** [AtomLinter/linter-jscs#122]
Using the linked [sample](http://www.filedropper.com/sample_11) I discovered that `linter-jscs` reports incorrect column position inside Atom. When I run `jscs` on provided `sample.js` file in console I get
![console](https://cloud.githubusercontent.com/assets/5924185/10560727/4135a262-7514-11e5-9d48-5835ac89a9b8.png)
which is correct, however inside Atom the issue is reported as
![atom1](https://cloud.githubusercontent.com/assets/5924185/10560733/550a7d76-7514-11e5-81d0-d8c141e2ce93.png)
error and line are correct, but column within line is not - it's not 7.
![atom2](https://cloud.githubusercontent.com/assets/5924185/10560755/cb7d1a4a-7514-11e5-8efe-ec76d6705e70.png)

**Comment by Arcanemagus** [comment]
For some reason this package decides to ignore the column specified by the linter and make one up of their own [here](https://github.com/AtomLinter/linter-jscs/blob/master/index.js#L119-L120).

`jscs` itself has the correct column as you can see from the output here:
```
JSON.stringify(errors)
"[{"filename":"C:\\Users\\Landon\\Downloads\\sample\\sample.js","rule":"requireSpaceBeforeBinaryOperators","message":"Operator + should not stick to preceding expression","line":3,"column":35,"fixed":true}]"
```

@iam4x or @simonappelt  do either of you know why this was done? I personally don't use this linter.

**Comment by casesandberg** [comment]
It looks like it was done like that so that the line with the error would be underlined with the dotted red. I think atom does a good job with the red dots to the left so I am going to fix the code such that that echos the proper column number.

**Comment by casesandberg** [comment]
This has been fixed! Go ahead and update to `3.4.0`
